### ISSUE TITLE
Version switching does not work.

### ISSUE THREAD

**Issue by ghost** [pink-mist/sbotools#86]
What I am doing is:

```
sboconfig -V 15.0
sbosnap update
```

This, seemingly, does some network operations, but the repo turns out to be the -current one, after all.
Even if I delete the `repo` directory, it seems to, still, pull the -current repo.

**Comment by drgibbon** [comment]
Yeah sbotools need some maintenance love. But you can also just manually set the value of `REPO` in `/etc/sbotools/sbotools.conf`. I have `REPO=https://gitlab.com/SlackBuilds.org/slackbuilds.git`

**Comment by drgibbon** [comment]
Or alternatively `sboconfig -r your_preferred_repo`
